Yatima/Compiler/Utils.lean

```diff
@@ -1,5 +1,5 @@
 import Lean
-import Yatima.ForStdLib
+import YatimaStdLib.RBTree
```

Comment:
How is it working? We haven't added the dependency in the lakefile. Is it pulling `YatimaStdLib` transitively?

Reply:
Yes. It goes like `Ipld <- StdLib <- LSpec`

Reply:
Hmmm, still I think we should explicitly add it as a dependency to make it less mysterious

Reply:
Ok, should we add LSpec too then?

Reply:
👍🏼